pysparkle/pysparkle/data_quality/data_quality_utils.py

```diff
@@ -0,0 +1,139 @@
+"""
+Data_quality utility functions to set up validations
+"""
+from typing import List, Dict
+from pyspark.sql import DataFrame
+
+from great_expectations.core.batch import RuntimeBatchRequest
+from great_expectations.core.expectation_configuration import (
+    ExpectationConfiguration)
+from great_expectations.core.expectation_validation_result import ExpectationSuiteValidationResult
+from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.data_context import BaseDataContext
+from great_expectations.data_context.types.base import DataContextConfig, \
+    FilesystemStoreBackendDefaults
+
+
+def create_datasource_config(datasource_name: str) -> BaseDataContext:
+    """
+    Given a string containing the datasource name, this function generates a data context instance
+
+    Args:
+        datasource_name (str): Name of the datasource to be validated
+
+    Returns:
+        BaseDataContext: populated data context instance to which expectations can be added
```

Comment:
```suggestion
        Populated data context instance to which expectations can be added
```
Type is already given as a hint in the function definition.